src/stories/pagefold/pagefold.scss

```diff
@@ -42,5 +46,13 @@ $pagefold-map: (
     position: absolute;
     top: 0;
     left: 0;
+
+    &.success {
+      background-color: $c-signal-success;
+    }
+
+    &.alert {
+      background-color: $c-signal-alert;
+    }
```

Comment:
If we are to follow the BEM structure of the project and reduce nesting I would suggest that we change this to:

```scss
.pagefold-triangle--success {
  background-color: $c-signal-success;
}

.pagefold-triangle--alert {
  background-color: $c-signal-alert;
}
```

Reply:
This makes total sense. A class that is just called .success is too generic. I fixed it 😸